fhir-server/src/main/java/com/ibm/fhir/server/filter/rest/FHIRRestServletFilter.java

```diff
@@ -134,6 +134,15 @@ public void doFilter(HttpServletRequest request, HttpServletResponse response, F
                 if (tenantKey != null) {
                     context.setTenantKey(tenantKey);
                 }
+
+                String type = dsPG.getStringProperty("type", null);
+                if (type != null) {
+                    // Based on the default for the database type, the code.
+                    Boolean enabled =
+                            FHIRConfigHelper.getBooleanProperty("multitenant",
```

Comment:
should this be `String dsPropertyName = FHIRConfiguration.PROPERTY_DATASOURCES + "/" + dsId + "/multitenant";`?

Reply:
where `dsId` is the "datastoreId" which can be looked up from the request context

Reply:
right. this is the pattern from the XA Datasource

Reply:
resolved